database/forms.py

```diff
@@ -56,6 +57,23 @@ class UserForm(CrispyModelForm):
         last_name = CharField(max_length=256)
         email = EmailField()
 
+        def clean(self):
+            cleaned_data = super().clean()
+            last_name = cleaned_data.get("last_name")
+            first_name = cleaned_data.get("first_name")
+            email = cleaned_data.get("email")
+            if any(i.isdigit() for i in first_name):
```

Comment:
statements after the first should be elifs

Reply:
sure